cgi-bin/LJ/Talk.pm

```diff
@@ -2256,6 +2256,40 @@ sub init_s2journal_js {
         ) ) if $opts{lastn};
 }
 
+# convenience function for keyboard shortcuts
+# args: $remote and $p for adding javascript to header
+# returns: nothing, just calls need_res
+sub init_s2journal_shortcut_js {
+    my ( $remote, $p ) = @_;
+
+    if ( $remote ) {
```

Comment:
nit: To prevent overly deep nesting, prefer to just return early without `$remote`.